pwiz_tools/Skyline/TestRunner/Program.cs

```diff
@@ -439,6 +444,9 @@ static int Main(string[] args)
 
             if (commandLineArgs.ArgAsBool("wait"))
             {
+                GC.Collect();

```

Comment:
What exactly is this "GC.Collect" for?
If doing a garbage collection improves things, it might mean we're doing something wrong somewhere else.
Was there a particular test that you added this for? Was there a particular file on disk that was being locked?

The last time I had to debug something like this, the "pending finalizers" list in dotMemory was very helpful.

Reply:
This is for the "wait until user presses a button" step at the end of TestRunner execution. I used it for leak finding in a test in a loop and collecting a snapshot at the end before the process exits. It doesn't have anything to do with the coverage feature though.